corehq/apps/linked_domain/applications.py

```diff
@@ -129,6 +139,7 @@ def unlink_app(linked_app):
     app_copy.save()
 
     # remove linked app
-    linked_app.delete()
+    full_linked_app = get_app(linked_app.domain, linked_app._id)
```

Comment:
We use full apps much more often than briefs. Current naming conventions are that `app` or `linked_app` will refer to an actual app, and then brief apps use `brief` in their identifier.

Reply:
Good to know.